scripts/hak/copy.js

```diff
@@ -53,13 +53,46 @@ async function copy(hakEnv, moduleInfo) {
                 err ? reject(err) : resolve(files);
             });
         });
-        for (const f of files) {
-            console.log("\t" + f);
-            const src = path.join(moduleInfo.moduleBuildDir, f);
-            const dst = path.join(moduleInfo.moduleOutDir, f);
 
-            await mkdirp(path.dirname(dst));
-            await fsProm.copyFile(src, dst);
+        if (moduleInfo.moduleBuildDirs.length > 1) {
+            if (!hakEnv.isMac()) {
+                console.error(
+                    "You asked me to copy multiple targets but I've only been taught " +
+                    "how to do that on a mac.",
```

Comment:
Feel free to ignore if you don't care...

```suggestion
                    "how to do that on macOS.",
```